static/css/main.css

```diff
@@ -113,3 +146,12 @@
     width: 644px;
 }
 /* Create Worklists CSS ends */
+
+
+/* User Worklists CSS starts */
```

Comment:
No need to do it now, but it likely makes sense to break these different sections into different files to keep things organized.

A nice rule of thumb is that whenever there's a comment like that, dividing code in sections within a single unit (file/function/method/whatever), that's a good candidate to be moved to a separate unit.

Reply:
Done. I've split it into components based on different HTML it corresponds to.